### PR TITLE
Fix more errors

### PR DESCRIPTION
**Pull request by NycuRO**

### CODE REVIEW

File: src/pocketmine/command/defaults/GamemodeCommand.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ public function execute(CommandSender $sender, $currentAlias, array $args){
 				Command::broadcastCommandMessage($sender, new TranslationContainer("commands.gamemode.success.self", [' ', ' ', Server::getGamemodeString($gameMode)]));
 			}else{
 				$target->sendMessage(new TranslationContainer("gameMode.changed"));
-				Command::broadcastCommandMessage($sender, new TranslationContainer("commands.gamemode.success.other", [' ', [$target->getName(), Server::getGamemodeString($gameMode)]));
+				Command::broadcastCommandMessage($sender, new TranslationContainer("commands.gamemode.success.other", [' '], [$target->getName(), Server::getGamemodeString($gameMode)]));
```
Comment (fuyutsuki) [inline src/pocketmine/command/defaults/GamemodeCommand.php]: Syntax error, not necessary.

Comment (fuyutsuki) [inline src/pocketmine/command/defaults/GamemodeCommand.php]: The second argument
`[' '], [$target->getName(), Server::getGamemodeString($gameMode)]`
↓
`[' ', $target->getName(), Server::getGamemodeString($gameMode)]`

**Comment by AppleDevelops** [comment]
Please resolve the conflics

**Comment by NycuRO** [comment]
@AppleDevelops You can fix?

**Comment by Nought57** [comment]
I think "Fix duplicated chat message" is a Required Commit.

**Comment by Nought57** [comment]
I think "Fix duplicated chat message" is a Required Commit.

**Pull request closed without merging**